npm/build-docs.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+// ----------------------------------------------------------------------------------------------------------------------
+// This script is intended to generate documentation of this module
+// ----------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+require('colors');
+
+// Stop on first error
+set('-e');
+
+console.log('Generating documentation...'.yellow.bold);
+
+// clean directory
+test('-d', './out/docs') && rm('-rf', './out/docs');
+
+exec('node node_modules/.bin/jsdoc -c .jsdoc-config.json -u docs/ lib/*');
```

Comment:
This is because ShellJS does not correctly pick up the module path, right? 
Because I _think_ that things work fine even without `node_modules/.bin/<whatever>`